### ISSUE TITLE
Windows 10 Start menu should be floating/ignored by default.

### ISSUE THREAD

**Issue by yzhuang** [rickbutton/workspacer#101]
Current behavior: when the Windows key is pressed, the start menu pops up.  This start menu (more specifically, the "type here to search" searchbox) is treated as a new window managed by workspacer--i.e. it cause other existing windows to shrink.  However, workspacer does not seem to be able to manage (resize or reposition) the Windows start menu window anyway. 

Expected behavior: workspacer should ignore the start menu.

After reading the docs, I believe it can be achieved by something similar to 
```
context.WindowRouter.AddFilter((window) => !window.Title.Contains("foobar"));
```

However, filing a request here because:
1. I couldn't figure out the title of the Windows start menu.
2. I believe that this behavior should be default to benefit everyone.

Thank you!

**Comment by JCallicoat** [comment]
Looking at a window dump of the start menu in the debug console shows these properties:

```
window.Handle                    = 65966
window.Title                     = "Start"
window.Class                     = "Windows.UI.Core.CoreWindow"
window.ProcessId                 = 11232
window.ProcessName               = "StartMenuExperienceHost"
window.ProcessFileName           = "--NA--"
```

So this commit should already filter it from window routing https://github.com/rickbutton/workspacer/commit/7cc16f334ce9263a9147600b583d63c89797c4c8. However, that just prevents it from being routed to a different workspace, so it will always appear on the focused workspace, which is different than what you want.

I think you are asking for workspacer to ignore laying out the window entirely like the alt-tab window.

**Comment by JCallicoat** [comment]
It looks like the actual `ProcessName` of the window where you type is "SearchApp". Does this work for you?

```
context.WindowRouter.AddFilter((window) => window.ProcessName != "SearchApp");
```

You can also write it with the helper method

```
context.WindowRouter.IgnoreProcessName("SearchApp");
```

**Comment by rickbutton** [comment]
This was fixed on master, but I hadn't gotten around to releasing a new version since this repo was fairly dormant. This will be fixed in `0.9.8`! Thanks for opening the issue!

**Comment by rickbutton** [comment]
`0.9.8` is released! workspacer should automatically prompt for the update.
